packages/react-ui-core/src/Modal/Modal.js

```diff
@@ -103,9 +115,14 @@ export default class Modal extends PureComponent {
       CloseButton,
       hasOverlay,
       closeOnOverlayClick,
+      lockBodyScrolling,
       ...props
     } = this.props
 
+    if (lockBodyScrolling) {
+      this.toggleBodyClass(true)
```

Comment:
Is this something that should happen inside the render method or can we move it to a lifecycle method?

Reply:
I could move it to `componentDidMount`, I think.

Reply:
Good idea. Otherwise the DOM will try to update the document root classlist every single time React renders.

Reply:
Yup, that would make more sense, since it seems it should only happen once